path_config.go

```diff
@@ -13,6 +14,7 @@ import (
 const (
 	configPath            = "config"
 	defaultPasswordLength = 64
+	defaultSchema         = "openldap"
```

Comment:
Note that this will break existing beta testers. That's fine (we don't promise backwards compat), but something to note for docs/changelog/etc.

Reply:
Will do, thanks!